Gems/GraphModel/Code/Source/Model/GraphContext.cpp

```diff
@@ -36,7 +38,7 @@ namespace GraphModel
                 ->Attribute(AZ::Script::Attributes::Module, "editor.graph")
                 ->Method("GetSystemName", &GraphContext::GetSystemName)
                 ->Method("GetModuleFileExtension", &GraphContext::GetModuleFileExtension)
-                ->Method("GetAllDataTypes", &GraphContext::GetAllDataTypes)
+                //->Method("GetAllDataTypes", &GraphContext::GetAllDataTypes) // GHI-15121 Unbinding until asserts with AP behavior context reflection are resolved
```

Comment:
This should be fixed by this PR changes, forgot to uncomment it and remove the comment?

Reply:
This is the only function that is problematic. No other errors or asserts are being reported. I'll comment this out for now, since it's not being used in script or tests yet, and uncomment once there's more time to investigate.